docs/API/Release_Notes/API_Release_Notes.md

```diff
@@ -37,6 +38,25 @@
 | [v1.1.0](API_Release_Notes.md#v110) | May 25, 2016 |
 | [v1.0.1](API_Release_Notes.md#v101) | May 16, 2016 |
 
+## v7.4.2
+
+* __GDC Product__: Application Programming Interface (API)
+* __Release Date__:  July 30, 2024
+
+### New Features and Changes
+
+* The API indices will include fields from the "other clinical attribute" entity types. This will be visible in the next data release.
```

Comment:
Should the entity name be capitalized, i.e. "Other Clinical Attribute" and/or should we link to the entity in the Data Dictionary Viewer ([link](https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=other_clinical_attribute)) to show users what we are talking about?

Change "This" > "Data for these fields"?